backend/src/backend/core.clj

```diff
@@ -1,11 +1,15 @@
 (ns backend.core
   (:require [org.httpkit.server :refer [run-server]]))
 
-(defn app [req]
+(defn app 
+  "Says hello, world"
+  [req]
   {:status  200
    :headers {"Content-Type" "text/html"}
    :body    (str "Hello, World")})
 
-(defn -main [& args]
+(defn -main 
```

Comment:
Hvorfor er det -main og ikke bare main?

Reply:
https://stackoverflow.com/questions/4690758/splitting-a-clojure-namespace-over-multiple-files/4692067#4692067

Jeg fulgte en tutorial™ som gjorde det sånn.

Reply:
Ser ut som det er en konvensjon for private funksjoner. https://stackoverflow.com/questions/10846423/is-there-a-clojure-convention-for-naming-private-functions